server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -494,22 +494,29 @@ private static void flushAll(final ClientContext context) {
     try {
       flusher.join(3000);
     } catch (InterruptedException e) {
-      // ignore
+      log.warn("Interrupted while waiting to join Flush thread");
```

Comment:
I don't see much reason to suppress the interrupt flag here, so we should restore it in case there's something monitoring it. This is the recommended best practice if we're not re-throwing it.

```suggestion
      Thread.currentThread().interrupt();
      log.warn("Interrupted while waiting to join Flush thread", e);
```

Reply:
Yeah, I didn't include it because it's likely called from the CLI. I can add it though.

Reply:
It could be called by a script, too. Even if it's called by a user from the CLI, I don't think it's that bad to show them the mess they caused when they hit Ctrl-C :smiley_cat: